tpcc/worker.go

```diff
@@ -166,5 +197,11 @@ func (w *worker) run(errCh chan<- error, wg *sync.WaitGroup) {
 		if *maxOps > 0 && v >= *maxOps {
 			return
 		}
+
+		thinkTime := -math.Log(rand.Float64()) * float64(t.thinkTime)
+		if thinkTime > (t.thinkTime * 10) {
+			thinkTime = t.thinkTime * 10
```

Comment:
Where is the `* 10` coming from?

Reply:
It's part of the spec. I will add a comment. The think time distribution "may be truncated at 10 times its mean value".